conda/miniforge-init.sh

```diff
@@ -1,13 +0,0 @@
-# >>> conda initialize >>>
-__conda_setup="$('$HOME/miniforge3/bin/conda' 'shell.bash' 'hook' 2> /dev/null)"
-if [ $? -eq 0 ]; then
-    eval "$__conda_setup"
-else
-    if [ -f "$HOME/miniforge3/etc/profile.d/conda.sh" ]; then
-        . "$HOME/miniforge3/etc/profile.d/conda.sh"
-    else
-        export PATH="$HOME/miniforge3/bin:$PATH"
-    fi
-fi
```

Comment:
If this is changing, then we will need to do something like `source ~/.mambaforge-init.sh` instead of what we do at present. Surely this crops up somewhere in the docs?

Reply:
yeah true, I'll find that and change it too